Source/SwiftLintFramework/Rules/MissingDocsRule.swift

```diff
@@ -54,17 +54,9 @@ extension File {
 }
 
 public enum AccessControlLevel: String {
-    case Private = "private"
```

Comment:
Actually the distinction between `rawValue` and `sourcekitValue()` was intentional here. We sometimes use `AccessControlLevel.description` in violation messages (I think).